source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,179 @@
+
+After the paint store runs this query, they find that orders with
+``_id`` "2" and "3" contain more than three, but less than ten cans
+of paint.
+
+For a runnable ``find()`` example, see our :doc:`Find Multiple
+Documents </usage-examples/find>` page. 
+
+.. _retrieve-aggregate:
+
+Aggregate
+~~~~~~~~~
+
+Use the aggregate operation to perform an aggregation pipeline.  An
+aggregation pipeline is a multi-staged transformation that produces an
+aggregated result. To perform transformations, you can define in stages
+how to match documents, rename fields and group values. 
+
+To perform an aggregate operation, use the ``aggregate()`` method on a
+collection. This method accepts aggregation expressions to run in
+sequence. For more information on aggregation, see our :doc:`Aggregation
+</fundamentals/aggregation>` guide.
+
+Example
+```````
+
+The paint store would like to know which color is the most purchased.
+
+To address the scenario, the paint store needs to create an aggregation
+pipeline that:
+
+- matches all the documents in the ``paint_orders`` collection
+- groups orders by colors 
+- sums up the quantity field by color
+- sorts the results in descending order by quantity
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Retrieve.java
+   :language: java
+   :dedent:
+   :start-after: begin aggregateExample
+   :end-before: end aggregateExample
+
+The following shows the output of the query specified above:
+
+.. code-block:: json
+
+    { "_id": "green", "qty": 19 }
+    { "_id": "purple", "qty": 14 }
+
+After the paint store runs the query, they find that "green" is the most
+purchased color.
+
+For more information on how to construct an aggregation pipeline, see
+the MongoDB server manual page on :manual:`Aggregation </aggregation>`.
+
+.. _retrieve-watch:
+
+Change Streams
+~~~~~~~~~~~~~~
+
+Open a change stream to monitor real-time changes to a collection.
+Change streams subscribe to all data changes specified and 
+immediately react to them. For more information on change streams, see
+the MongoDB server manual page on :manual:`Change Streams </changeStreams>`. 
+
+To open a change stream, use the ``watch()`` method on a
+``MongoCollection``, ``MongoDatabase`` or ``MongoClient``. You can pass
+this method to an aggregation pipeline to notify you and react as soon as a
+change occurs.
+
+Example
+```````
+
+The paint store would like notifications for when customers send in new
+orders or update their order.
+
+To address this scenario, the paint store needs to:
+
+- create an aggregation pipeline that filters for "insert" and "update" as a``operationType``
+- create a watch stream on the aggregation pipeline
+- print changes picked up by the watch stream
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Retrieve.java
+   :language: java
+   :dedent:
+   :start-after: begin watchExample
+   :end-before: end watchExample
+
+After the paint store runs the query, they will get notifications for any
```

Comment:
Two items:
1) avoid future tense
2) is creating a change stream the same as "runs a query?" or should they be different?